docs/quickstart/setup_quickstart.md

```diff
@@ -72,6 +72,15 @@ $ cd build-target               # this is where Flink is installed to
 ~~~
 {% endif %}
 
+### Alternatively
```

Comment:
What if we renamed "Alternatively" to "Install on MacOS".

And perhaps pull the prior "Download" into the if/else and rename to "Download and Unpack" and "Download and Compile"?